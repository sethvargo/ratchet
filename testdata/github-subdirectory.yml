jobs:
  my_job:
    runs-on: 'ubuntu-latest'

    container:
      image: 'ubuntu:20.04'

    services:
      nginx:
        image: 'nginx:1.21'

    steps:
      - uses: 'actions/checkout@v3'

      - uses: 'abcxyz/github-token-minter/.github/actions/mint-token@268f5db75d6ce3da57eb82ee513aa0fe7231d89b' # ratchet:abcxyz/github-token-minter@v0.0.24

      - uses: 'docker://ubuntu:20.04'
        with:
          uses: '/path/to/user.png'
          image: '/path/to/image.jpg'

      - runs: |-
          echo "Hello ðŸ˜€"
          if [ "true" == "false" ];
            echo "NOPE"
          fi

  other_job:
    uses: 'my-org/my-repo/.github/workflows/my-workflow.yml@v0'

  final_job:
    uses: './local/path/to/action'
