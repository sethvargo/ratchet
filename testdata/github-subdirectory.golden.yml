jobs:
  my_job:
    runs-on: 'ubuntu-latest'

    container:
      image: 'ubuntu:20.04'

    services:
      nginx:
        image: 'nginx:1.21'

    steps:
    - uses: 'actions/checkout@v3'

    - uses: 'abcxyz/github-token-minter/.github/actions/mint-token@268f5db75d6ce3da57eb82ee513aa0fe7231d89b' # ratchet:abcxyz/github-token-minter@v0.0.24

    - uses: 'docker://ubuntu:20.04'
      with:
        uses: '/path/to/user.png'
        image: '/path/to/image.jpg'

    - runs: |-
        echo "Hello ðŸ˜€"
        if [ "true" == "false" ];
          echo "NOPE"
        fi

  final_job:
    uses: './local/path/to/action'
